lib/ansible/modules/network/meraki/meraki_static_route.py

```diff
@@ -0,0 +1,243 @@
+    name:
+        description:
+        - Descriptive name of the static route.
+    subnet:
+        description:
+        - CIDR notation based subnet for static route.
+    gateway_ip:
+        description:
+        - IP address of the gateway for the subnet.
+    route_id:
+        description:
+        - Unique ID of static route.
+    fixed_ip_assignments:
+        description:
+        - List of fixed IP assignments for DHCP.
+        type: list
+    reserved_ip_ranges:
+        description:
+        - List of IP ranges reserved for static IP assignments.
+        type: list
+
+
+author:
+    - Kevin Breit (@kbreit)
+extends_documentation_fragment: meraki
+'''
+
+EXAMPLES = r'''
+- name: List all networks associated to the YourOrg organization
+  meraki_network:
+    auth_key: abc12345
+    state: query
+    org_name: YourOrg
+  delegate_to: localhost
+- name: Query network named MyNet in the YourOrg organization
+  meraki_network:
+    auth_key: abc12345
+    state: query
+    org_name: YourOrg
+    net_name: MyNet
+  delegate_to: localhost
+- name: Create network named MyNet in the YourOrg organization
+  meraki_network:
+    auth_key: abc12345
+    state: present
+    org_name: YourOrg
+    net_name: MyNet
+    type: switch
+    timezone: America/Chicago
+    tags: production, chicago
+  delegate_to: localhost
+'''
+
+RETURN = r'''
+data:
+    description: Information about the created or manipulated object.
+    returned: info
+    type: complex
+    contains:
+      net_id:
+        description: Identification string of network.
+        returned: success
+        type: string
+        sample: N_12345
+      net_name:
+        description: Written name of network.
+        returned: success
+        type: string
+        sample: YourNet
+      name:
+        description: Name of static route.
+'''
+
+import os
+from ansible.module_utils.basic import AnsibleModule, json, env_fallback
+from ansible.module_utils.urls import fetch_url
+from ansible.module_utils._text import to_native
+from ansible.module_utils.network.meraki.meraki import MerakiModule, meraki_argument_spec
+
+
+def get_static_routes(meraki, net_id):
+    path = meraki.construct_path('get_all', net_id=net_id)
+    r = meraki.request(path, method='GET')
+    return r
+
+
+def get_static_route(meraki, net_id, route_id):
+    path = meraki.construct_path('get_one', net_id=net_id, custom={'route_id': meraki.params['route_id']})
+    r = meraki.request(path, method='GET')
+    return r
+
+
+def main():
+
+    # define the available arguments/parameters that a user can pass to
+    # the module
+
+    argument_spec = meraki_argument_spec()
+    argument_spec.update(
+        net_id=dict(type='str'),
+        net_name=dict(type='str'),
+        name=dict(type='str'),
+        subnet=dict(type='str'),
+        gateway_ip=dict(type='str'),
+        state=dict(type='str', choices=['absent', 'query', 'present'], default='present'),
```

Comment:
Sort alphabetically, add defaults before choices.
```suggestion
        state=dict(type='str', default='present', choices=['absent', 'present', 'query']),
```